models/articles.models.js

```diff
@@ -24,6 +33,7 @@ const fetchArticles = (sort_by = "created_at", order = "desc") => {
     "comments_count",
   ];
   const validOrderValues = ["asc", "desc"];
+  const validTopics = ["mitch", "cats", "paper"];
```

Comment:
You can't check these like this because the production database has different topics in it - also what if more get added? 

You need to make a query to the database to check if the topic exists or not in the database you are in!